docs/UserGuide.md

```diff
@@ -68,15 +69,72 @@ not be so obvious!
 1. Refer to the [Features](#features) below for details of each command.
 
 --------------------------------------------------------------------------------------------------------------------
+## 3. About <a name="about"></a>
 
+### 3.1 Structure of this document
+
+We have structured this User Guide in a way so that you can find what you need easily and quickly. In the next section,
+[Section 3.2 - Reading this document](#reading-this-document), you can find sevaral useful tips on how to read this guide. The following section,
+[Section 4 - Features](#features), documents the four main features in **tCheck**, namely:
+
+   * Sales Tracking
+
+   * Ingredients Tracking
+
+   * Employee Management
+
+   * Other general features
+
+### 3.2 Reading this document <a name="reading-this-document"></a>
+
+This section introduces you to some key technical terms, symbols and syntax that are used throughout this guide.
+You may want to familarize yourself with them before moving on to the next section.
+
+#### 3.2.1 Terminology related to the GUI
+
+Figure 2 shows the GUI of **tCheck**, annotated with descriptions for all components of the GUI.
+
+<img src="images/tCheckInfographic.png" />
```

Comment:
I'm not sure why, but I couldn't view the image when I was looking at this document in your branch. Did you remember to upload the image?

Reply:
I double checked and not sure why this is happening as well. To be safe, I will just delete the current one and reupload, thanks